src/riak_kv_entropy_manager.erl

```diff
@@ -418,7 +453,12 @@ reload_hashtrees(Ring, State=#state{trees=Trees}) ->
     Existing = dict:from_list(Trees),
     MissingIdx = [Idx || Idx <- Indices,
                          not dict:is_key(Idx, Existing)],
-    _ = [riak_kv_vnode:request_hashtree_pid(Idx) || Idx <- MissingIdx],
```

Comment:
This is left over from a WIP commit. I was originally doing some logic in the special empty list case before returning ok but I removed it and forgot to revert this as well.